tests/mocks/components.py

```diff
@@ -78,6 +83,29 @@ async def is_moving(self) -> bool:
         return not self.is_stopped
 
 
+class MockAudioInput(AudioInput):
+    def __init__(self, name: str, properties: AudioInput.Properties):
+        super().__init__(name)
+        self.properties = properties
+
+    async def stream(self, **kwargs) -> AudioStream:
+        async def read() -> AsyncIterator[Audio]:
+            for i in range(10):
+                yield Audio(
+                    AudioChunkInfo(
+                        sample_format=SampleFormat.SAMPLE_FORMAT_FLOAT32_INTERLEAVED,
```

Comment:
Is it worth testing against other formats as well?

Reply:
Not really since this doesn't actually produce audio, just a fake set of bytes that I can test against